appendix-basic65.tex

```diff
@@ -10823,7 +10956,7 @@ \subsection{WPOKE}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$FE \$1D
 \item [Format:] {\bf WPOKE} address{\bf,} word [{\bf,} word ...]
-\item [Usage:]  Writes one or more words into memory
+\item [Usage:]  Writes one or more 16-bit words into memory
                 or memory mapped I/O, starting at {\bf address}.
```

Comment:
Is it worth mentioning that the MEGA65's POKE/WPOKE differ slightly to the C64's POKE,  in that they don't produce an "?ILLEGAL QUANTITY ERROR" if the poked value is out of range (e.g., higher than 255 for poke, higher than 65535 for wpoke). This was a capability Bit Shifter added.

Bit shifter cropped/AND'ed the value with some max value mask (e.g. for poke, a value of 256=0, 257=1)

Reply:
This is already mentioned in POKE: "If {\bf value} is in the range of 0 -- 255, this is poked into memory, otherwise the low byte of value is used." (Not quite how I would have phrased it, but it's mentioned. :) ) I added a similar fact here for completeness.

Reply:
Ah nice, though now that you've directed my eyeballs to the POKE command's text, I noticed it wrote there:

> because POKE uses the lowy byte anyway.

Was this 'lowy' a typo of 'low'?